src/main/java/com/github/manolo8/darkbot/core/utils/EntityList.java

```diff
@@ -128,6 +129,24 @@ private void onEntityCreate(Entity entity) {
         this.eventBroker.sendEvent(new EntityCreateEvent(entity));
     }
 
+    public FakeEntity.FakeMine createFakeMine(int typeId, eu.darkbot.api.game.other.Location  loc, long removeDistance, long keepAlive) {
```

Comment:
```suggestion
    public FakeEntity.FakeMine createFakeMine(int typeId, eu.darkbot.api.game.other.Location loc, long removeDistance, long keepAlive) {
```
minor formatting change